### ISSUE TITLE
[Markers and Paths] Feature Request: Transparency sliders for Fullscreen/Minimap

### ISSUE THREAD

**Issue by astralarya** [blish-hud/Community-Module-Pack#125]
Could sliders to adjust the transparency of markers and trails on the fullscreen map and minimap be added?

**Comment by dlamkins** [comment]
Yes.  In fact, it's on our road-map.  I'm going to move this to our pathing module repo for better tracking, just FYI.

I'll update here once we have that implemented and available for use.  🙂
